EncoreTickets.SDK.Tests/IntegrationTests/ContentServiceTests.cs

```diff
@@ -37,12 +44,40 @@ public void GetProducts_Successful()
 
             foreach (var product in products)
             {
-                Assert.False(string.IsNullOrEmpty(product.Name));
-                Assert.False(string.IsNullOrEmpty(product.Id));
-                if (product.Venue != null)
-                {
-                    Assert.False(string.IsNullOrEmpty(product.Venue.Name));
-                }
+                AssertProduct(product);
+            }
+        }
+
+        [Test]
+        public void GetProductById_Successful()
+        {
+            var productId = configuration["Content:TestProductId"];
+
+            var product = service.GetProductById(productId);
+
+            AssertProduct(product);
+        }
+
+        [Test]
+        public void GetProductById_Exception404()
+        {
+            var productId = "invalid";
+
+            var exception = Assert.Catch<ApiException>(() =>
+            {
+                var product = service.GetProductById(productId);
+            });
+
+            Assert.AreEqual(HttpStatusCode.NotFound, exception.ResponseCode);
+        }
+
+        private void AssertProduct(Product product)
```

Comment:
Product is not a statement, you can't assert it. Maybe a better name would be AssertProductPropertiesAreSet or something like that?